src/components/sessionContext/SessionProvider.tsx

```diff
@@ -44,13 +44,13 @@ const SessionProvider: React.FC<{ children: React.ReactNode }> = ({ children })
   useQuery({
     queryKey: [WHO_AM_I_QUERY],
     refetchOnWindowFocus: !helpers.DEV_MODE,
-    queryFn: () => {
-      return axios.get(process.env.REACT_APP_USER_SERVICE_CURRENT || '').then(res => {
-        if (res.data.username) {
-          setSessionData(prev => ({ ...prev, authorized: true, username: res.data.username }));
-        }
-      });
-    }
+    // queryFn: () => {
+    //   return axios.get(process.env.REACT_APP_USER_SERVICE_CURRENT || '').then(res => {
+    //     if (res.data.username) {
+    //       setSessionData(prev => ({ ...prev, authorized: true, username: res.data.username }));
+    //     }
+    //   });
+    // }
```

Comment:
Given this is the "for real :tm:" repo, I think it is better to fix this before merging

comments on the original PR context https://github.com/quipucords/quipucords-ui-refresh/pull/7#discussion_r1423324285

Reply:
we haven't got to the auth/login stuff yet, i can put a todo here similar to your following comment